bip-0345.mediawiki

```diff
@@ -5,7 +5,8 @@
   Author: James O'Beirne <vaults@au92.org>
           Greg Sanders <gsanders87@gmail.com>
   Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0345
-  Status: Draft
+  Status: Withdrawn
+  Superseded-By: 443
```

Comment:
Please move this line below the Post-History header, the headers are [required to be in a specific order](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#bip-header-preamble).